.travis.yml

```diff
@@ -1,8 +1,8 @@
-osx_image: xcode8.3
+osx_image: xcode9
 language: objective-c
 podfile: Demo/Podfile
 cache: cocoapods
 before_install:
-  - gem install cocoapods -v '1.2.1'
+  - gem install cocoapods -v '1.3.1'
   - pod repo update
 script: cd Demo ; set -o pipefail && xcodebuild -workspace 'Demo.xcworkspace' -scheme 'Demo' -configuration 'Debug' -sdk iphonesimulator -destination platform='iOS Simulator',OS='10.3',name='iPhone 6s' build test  | xcpretty -c --test
```

Comment:
Hi @tomisacat, can you just update `OS='10.3',name='iPhone 6s'` to iOS11 and iPhone 8 to make sure travis passes?

Reply:
I've already fixed it on https://github.com/RxSwiftCommunity/RxNimble/pull/27, since it's pretty urgent. Thanks a lot for this man!

Reply:
Sorry about just noticing your comment, glad it helps!